openquake/server/settings.py

```diff
@@ -226,6 +226,17 @@
 if not os.environ.get('OQ_APPLICATION_MODE'):
     os.environ['OQ_APPLICATION_MODE'] = APPLICATION_MODE
 
+if APPLICATION_MODE.upper() in ('TOOLS_ONLY',):
```

Comment:
Please be aware that yesterday we made the APPLICATION_MODE case-sensitive (always capitalized) and we added a check to make sure that the specified application mode is included in the list of available ones.

Reply:
Fixed with this commit https://github.com/gem/oq-engine/pull/9565/commits/c62fb9812ac7fcfb7735361ec173b191b0efce85